resources/puppetlabs/lein-ezbake/template/global/ext/cli/start.erb

```diff
@@ -26,9 +35,10 @@ while [ "$cur" == "$initial" ] ;do
 
     ((timeout--))
     if [ "$timeout" = 0 ]; then
-        echo "Startup timed out after <%= EZBake::Config[:start_timeout] %> seconds"
+        echo "Startup timed out after <%= EZBake::Config[:start_timeout] %> seconds" >2
```

Comment:
This will create a file called 2. should be a redirection to stderr instead `1>&2` iirc.